api/api.go

```diff
@@ -3,35 +3,35 @@ package api
 import (
 	"context"
 
-	"github.com/ONSdigital/dp-table-renderer/health"
-	"github.com/ONSdigital/go-ns/log"
-	"github.com/ONSdigital/go-ns/server"
+	"github.com/ONSdigital/dp-healthcheck/healthcheck"
+	dphttp "github.com/ONSdigital/dp-net/http"
+	"github.com/ONSdigital/log.go/log"
 	"github.com/gorilla/handlers"
 	"github.com/gorilla/mux"
 
 	"net/http"
 )
 
-var httpServer *server.Server
+var httpServer *dphttp.Server
 
 // RendererAPI manages rendering tables from json
 type RendererAPI struct {
 	router *mux.Router
 }
 
 // CreateRendererAPI manages all the routes configured to the renderer
-func CreateRendererAPI(bindAddr string, allowedOrigins string, errorChan chan error) {
+func CreateRendererAPI(ctx context.Context, bindAddr string, allowedOrigins string, errorChan chan error, hc *healthcheck.HealthCheck) {
 	router := mux.NewRouter()
-	routes(router)
+	routes(router, hc)
 
-	httpServer = server.New(bindAddr, createCORSHandler(allowedOrigins, router))
+	httpServer = dphttp.NewServer(bindAddr, router)
 	// Disable this here to allow main to manage graceful shutdown of the entire app.
 	httpServer.HandleOSSignals = false
 
 	go func() {
-		log.Debug("Starting table renderer...", nil)
+		log.Event(ctx, "starting table renderer", log.INFO)
 		if err := httpServer.ListenAndServe(); err != nil {
```

Comment:
We are missing check for regular close error here. 

```suggestion
		if err := httpServer.ListenAndServe(); err != nil && err != http.ErrServerClosed {
```